.storybook/webpack.config.js

```diff
@@ -12,6 +12,7 @@ module.exports = async ({ config }) => {
     config.resolve.alias = {
         ...config.resolve.alias,
         'box-ui-elements-locale-data': path.resolve(`i18n/${language}`),
+        'box-locale-data': path.resolve(`node_modules/@box/cldr-data/locale-data/${language}`),
```

Comment:
We need to apply the same changes to `scripts/webpack.config.js`.

Reply:
Good catch! Missed that one. I've added it now.